lib/rdoc/options.rb

```diff
@@ -868,6 +876,12 @@ def parse argv
 
       opt.separator nil
 
+      opt.on("--plugins=PLUGINS", "-P", Array, "Use plugins") do |value|
+        @plugins.concat value
+      end
```

Comment:
We can use multiple `--plugin` to add multiple plugins.

```suggestion
      opt.on("--plugin=PLUGIN", "-P", "Use plugin") do |plugin|
        @plugins << plugin
      end
```

Reply:
I'm not sure which interface is better.

```sh
# This
rdoc --plugins yard_plugin,another_plugin

# Or that
rdoc --plugin yard_plugin --plugin another_plugin
```

Correct me if I'm wrong, but I think the later way is kind of rare.

Reply:
What does "rare" mean here?
The latter is used in many cases. For example, `ruby`'s `-I` is the latter interface. `curl`'s `--data`/`--header`/... are also the latter interface.

The latter style is easyer to use from a script:

```bash
args=()
if [ "$USE_X_PLUGIN" = "yes" ]; then
  args+=(--plugin X_plugin)
fi
if [ "$USE_Y_PLUGIN" = "yes" ]; then
  args+=(--plugin Y_plugin)
fi
rdoc "${args[@]}" ...
```

Reply:
@kou Thank you for correcting me, after some investigation, I noticed that we have both styles in Ruby and other tools.
Personally I don't have a strong opinion here. For the future version we can consider YAML configuration as a primary way to enable plugins (like [rubocop](https://docs.rubocop.org/rubocop/1.74/configuration.html)).
I'm not against using multiple `--plugin` option here.